src/pages/EmployeeListing/List/components/Pagination/hook/usePagination.ts

```diff
@@ -0,0 +1,88 @@
+import { useMemo } from 'react';
+
+export const DOTS = '...';
+
+const range = (start: number, end: number) => {
+    let length = end - start + 1;
+    return Array.from({ length }, (_, idx) => idx + start);
+};
+
+
+
+function usePagination({
+    totalPageCount,
+    pageSize,
+    siblingCount = 1,
+    currentPage
+}: {
+    totalPageCount: number,
+    pageSize: number,
+    siblingCount?: number,
+    currentPage: number
+}) {
+
+    const paginationRange = useMemo(() => {
+
+        // Pages count is determined as siblingCount + firstPage + lastPage + currentPage + 2*DOTS
+        const totalPageNumbers = siblingCount + 5; // always 6
```

Comment:
move these magic numbers to constants

Reply:
Resolved in 70b4934300f70439e5aca8f8c21bc6affc0a87e5